views/referenda/referendum.pug

```diff
@@ -0,0 +1,64 @@
+extends ../skeleton
+block title
+  title #{referendum.title}
+block breadcrumb
+  li
+    a(href=__('/referenda')) #{__('referenda')}
+  li
+    if isActive
+      a(href=__('/referenda/active')) #{__('referenda-active-title')}
+    else
+      a(href=__('/referenda/complete')) #{__('referenda-complete-title')}
+  li
+    a #{referendum.title}
+block content
+  .grid-row
+    .column-full
+      h1.heading-xlarge.title #{referendum.title}
+  .grid-row
+    .column-full
+        p.lede.
+            This is a referendum about "#{referendum.title}".
+        if referendum.binding
+            p.
```

Comment:
This would probably be a panel 
```suggestion
            p.panel.panel-border-wide
```
* See documentation in [GOV.UK Elements](https://govuk-elements.herokuapp.com/typography/#typography-inset-text) (the legacy library we're using here)
* See documentation in [GOV.UK Design System](https://design-system.service.gov.uk/components/inset-text/) (current GDS guidance)